mitxpro/templatetags/render_bundle.py

```diff
@@ -51,7 +51,7 @@ def _get_bundle(request, bundle_name):
 
 
 @register.simple_tag(takes_context=True)
-def render_bundle(context, bundle_name):
+def render_bundle(context, bundle_name, attrs=""):
```

Comment:
Can you rename this param to `added_attrs`, and add the param to the docstring with an explanation of what it does (ex: `Optional string of HTML attributes to add to the script/link tag`). `added_attrs` makes it a little bit clearer that you're not specifying _all_ attributes

Reply:
Oh, sorry, forgot to add the docstring earlier. Updated now.